WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/lists/sections/insights/usecases/TotalFollowersUseCase.kt

```diff
@@ -61,6 +65,11 @@ class TotalFollowersUseCase @Inject constructor(
         return items
     }
 
+    private fun addActionCard() {
+        val followers = summaryStore.getSummary(statsSiteProvider.siteModel)?.followers
+        if (followers != 0) actionCardHandler.dismiss(GROW)
```

Comment:
Function name is `add...` but it seems this function only `dismiss()` thing. Why do we dismiss in an add function?

Reply:
Reworked this part now, it is supposed to add actually.  Thanks